test/00_operators/ReductionTests.cu

```diff
@@ -225,6 +225,44 @@ TYPED_TEST(ReductionTestsNumericNoHalf, Sum)
       }
     }
   }
+  
+  {
+    tensor_t<TypeParam, 2> t2a({30, 40});
+    tensor_t<TypeParam, 2> t2b({30, 40});
+    
+    auto t4 = ones<TypeParam>({30, 40, 50, 60});
+
+    sum(t2a, t4, 0);
```

Comment:
You can remove the stream if you want to make it a little more readable when we point people here